.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@8764a52df183aa0ccea74521dfd9d506ffc7a19a # v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+        cache: maven
+    - name: Do update & install ffmpeg
+        run:|
+          sudo apt-get update
+          sudo apt-get -y install ffmpeg
```

Comment:
The lack of space before the `|` character is causing a syntax error. Also, I believe the `-y` flag is unnecessary.

Reply:
(unnecessary in the context of GitHub Actions)